UI/Views/Shared/_Layout.cshtml

```diff
@@ -45,7 +44,8 @@
     </footer>
     <script src="~/lib/jquery/dist/jquery.min.js"></script>
     <script src="~/lib/bootstrap/dist/js/bootstrap.bundle.min.js"></script>
-    <script src="~/js/site.js" asp-append-version="true"></script>
+    <script src="~/lib/microsoft/signalr/dist/browser/signalr.js"></script>
```

Comment:
I used [this](https://learn.microsoft.com/en-us/aspnet/core/tutorials/signalr?view=aspnetcore-8.0&WT.mc_id=dotnet-35129-website&tabs=visual-studio#add-signalr-client-code) microsoft example. Do you know what is the difference?

Reply:
![image](https://github.com/user-attachments/assets/3979ac63-32aa-4758-93cb-b6ab0a8f2411)

See above image as its says we need to execute 2 cli command so insead of running that cli command i prefer [link](https://learn.microsoft.com/en-gb/aspnet/core/tutorials/signalr?view=aspnetcore-8.0&tabs=visual-studio#create-a-web-app-project)  folder structure